src/client/eppo-client.spec.ts

```diff
@@ -53,39 +43,37 @@ class TestConfigurationStore implements IConfigurationStore {
     return this._isInitialized;
   }
 }
-export async function init(configurationStore: IConfigurationStore) {
-  const axiosInstance = axios.create({
-    baseURL: 'http://127.0.0.1:4000',
-    timeout: 1000,
-  });
-
-  const httpClient = new HttpClient(axiosInstance, {
-    apiKey: 'dummy',
-    sdkName: 'js-client-sdk-common',
-    sdkVersion: packageJson.version,
-  });
 
+export async function init(configurationStore: IConfigurationStore) {
+  const httpClient = new FetchHttpClient(
+    'http://127.0.0.1:4000',
+    {
+      apiKey: 'dummy',
+      sdkName: 'js-client-sdk-common',
+      sdkVersion: '1.0.0',
+    },
+    1000,
+  );
   const configurationRequestor = new FlagConfigurationRequestor(configurationStore, httpClient);
   await configurationRequestor.fetchAndStoreConfigurations();
 }
 
 describe('EppoClient E2E test', () => {
-  const storage = new TestConfigurationStore();
+  global.fetch = jest.fn(() => {
```

Comment:
Jest runs each file in it's own process so I don't _think_ it's necessary to clean this up. But you may want to, just in case. Also not necessary, but consider putting it in `beforeAll()` just to cluster that one-time stuff together.

```ts
beforeAll(async () => {
  const original fetch = fetch;
  global.fetch = jest.fn(() => {
    ...
   });   
  await init(storage);
});

afterAll(()=> {
  global.fetch = originalFetch;
})
```

Update: I see you do it later on. I think you could just ax this one entirely and just save off and restore `fetch()` at the top level. Then have two nested describes for unobfuscated and obfuscated responses, where you mock `fetch()`